tests/test-utilities/mocks/query-stack-trace-error.ts

```diff
@@ -0,0 +1,22 @@
+import { Page } from '@playwright/test';
+import { apiUrl } from '~test-utilities/mock-apis';
+
+const json = {
+  queryResult: {
+    payloads: [
+      {
+        an: 'error',
+      },
+    ],
+  },
+  queryRejected: null,
+};
+
+export default async function mockQueryApiWithStackTraceError(page: Page) {
```

Comment:
@rossedfort is this `mocks` folder where you were thinking to add individual mock api utils? If not, I'll create a new one!

Reply:
Yep this looks good! FWIW - we could set a workflow up that returns this data when queried, but I think this is fine for now